org/teams.yaml

```diff
@@ -264,6 +264,11 @@ teams:
           - dhawton
           - kfaseela
           - mikemorris
+      Release Managers - 1.26:
+        description: Release managers for Istio 1.26
+        members:
+          - dhawton
+          - kfaseela
```

Comment:
What about the third person? Should we check with the TOC for the sign up?

Reply:
I raised in the previous TOC meeting to ensure they were aware. I poked again and was told a 3rd has not yet been found.

Reply:
I plan on re-raising in today's TOC meeting.